src/server/system_services/pool_server.js

```diff
@@ -460,12 +497,12 @@ async function scale_hosts_pool(req) {
         update: {
             pools: [{
                 _id: pool._id,
+                'hosts_pool_info.initialized': true,
```

Comment:
`initialized` should be set after scale succeeds (inside `scale_hosts_pool_job`) and not here